README.md

```diff
@@ -130,6 +130,10 @@ Alternatively, you can install PASEOS by using [pip](https://www.pypy.org/) as f
 cd PASEOS
 pip install -e .
 ```
+### Using Docker
+Two [Docker](https://www.docker.com/) images are available:
+* paseos: corresponding to the last release.
```

Comment:
We should link to them and provide a short description / or link to one how to install them I think?

Ideally we try out now to put it on dockerhub then I can test it with that?

Reply:
Yes, I also wanted to do that. We need to merge this PR before so we can test the full pipeline on main , put the images on Docker and then I will update README.

Reply:
I updated the README.